### ISSUE TITLE
1.4.0 Deck Error

### ISSUE THREAD

**Issue by rossedman** [spinnaker/spinnaker#1980]
*Deck Fails On 1.4.0 Upgrade*

*Cloud Provider*

- Kubernetes
- GCP Storage Backed

*Environment*

Spinnaker is managing multiple Kubernetes clusters in Azure. Backed by Google Storage. Running in distributed mode in Kubernetes. Also uses AAD for login/auth. 

*Feature Area*

Deck error.

*Description*

Once upgraded, not possible to login. Spinnaker just sits at splash screen.

*Steps to Reproduce*

Upgrade to 1.4.0, try to login. Tried to log out of AAD provider and clear cache. Error still present. 

*Additional Details*

Here are browser console logs

```
Uncaught SyntaxError: Unexpected identifier
app.js:209 Uncaught TypeError: Cannot read property 'feature' of undefined
    at Object.defineProperty.value (app.js:209)
    at __webpack_require__ (init.js:55)
    at Object.defineProperty.value (app.js:16169)
    at __webpack_require__ (init.js:55)
    at Object.<anonymous> (app.js:555)
    at __webpack_require__ (init.js:55)
    at Object.<anonymous> (app.js:794)
    at __webpack_require__ (init.js:55)
    at Object.<anonymous> (app.js:53988)
    at __webpack_require__ (init.js:55)
vendor.bundle.js:90337 Uncaught Error: [$injector:modulerr] Failed to instantiate module netflix.spinnaker due to:
Error: [$injector:nomod] Module 'netflix.spinnaker' is not available! You either misspelled the module name or forgot to load it. If registering a module ensure that you specify the dependencies as the second argument.
http://errors.angularjs.org/1.6.4/$injector/nomod?p0=netflix.spinnaker
    at http://myspinnaker.com/vendor.bundle.js:85483:12
    at http://myspinnaker.com/vendor.bundle.js:87679:17
    at ensure (http://myspinnaker.com/vendor.bundle.js:87600:38)
    at module (http://myspinnaker.com/vendor.bundle.js:87677:14)
    at http://myspinnaker.com/vendor.bundle.js:90313:22
    at forEach (http://myspinnaker.com/vendor.bundle.js:85820:20)
    at loadModules (http://myspinnaker.com/vendor.bundle.js:90297:5)
    at createInjector (http://myspinnaker.com/vendor.bundle.js:90219:19)
    at doBootstrap (http://myspinnaker.com/vendor.bundle.js:87331:20)
    at bootstrap (http://myspinnaker.com/vendor.bundle.js:87352:12)
http://errors.angularjs.org/1.6.4/$injector/modulerr?p0=netflix.spinnaker&p1=Error%3A%20%5B%24injector%3Anomod%5D%20Module%20'netflix.spinnaker'%20is%20not%20available!%20You%20either%20misspelled%20the%20module%20name%20or%20forgot%20to%20load%20it.%20If%20registering%20a%20module%20ensure%20that%20you%20specify%20the%20dependencies%20as%20the%20second%20argument.%0Ahttp%3A%2F%2Ferrors.angularjs.org%2F1.6.4%2F%24injector%2Fnomod%3Fp0%3Dnetflix.spinnaker%0A%20%20%20%20at%20https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A85483%3A12%0A%20%20%20%20at%20https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87679%3A17%0A%20%20%20%20at%20ensure%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87600%3A38)%0A%20%20%20%20at%20module%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87677%3A14)%0A%20%20%20%20at%20https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A90313%3A22%0A%20%20%20%20at%20forEach%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A85820%3A20)%0A%20%20%20%20at%20loadModules%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A90297%3A5)%0A%20%20%20%20at%20createInjector%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A90219%3A19)%0A%20%20%20%20at%20doBootstrap%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87331%3A20)%0A%20%20%20%20at%20bootstrap%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87352%3A12)
    at http://myspinnaker.com/vendor.bundle.js:85483:12
    at http://myspinnaker.com/vendor.bundle.js:87679:17
    at ensure (http://myspinnaker.com/vendor.bundle.js:87600:38)
    at module (http://myspinnaker.com/vendor.bundle.js:87677:14)
    at http://myspinnaker.com/vendor.bundle.js:90313:22
    at forEach (http://myspinnaker.com/vendor.bundle.js:85820:20)
    at loadModules (http://myspinnaker.com/vendor.bundle.js:90297:5)
    at createInjector (http://myspinnaker.com/vendor.bundle.js:90219:19)
    at doBootstrap (http://myspinnaker.com/vendor.bundle.js:87331:20)
    at bootstrap (http://myspinnaker.com/vendor.bundle.js:87352:12)
http://errors.angularjs.org/1.6.4/$injector/modulerr?p0=netflix.spinnaker&p1=Error%3A%20%5B%24injector%3Anomod%5D%20Module%20'netflix.spinnaker'%20is%20not%20available!%20You%20either%20misspelled%20the%20module%20name%20or%20forgot%20to%20load%20it.%20If%20registering%20a%20module%20ensure%20that%20you%20specify%20the%20dependencies%20as%20the%20second%20argument.%0Ahttp%3A%2F%2Ferrors.angularjs.org%2F1.6.4%2F%24injector%2Fnomod%3Fp0%3Dnetflix.spinnaker%0A%20%20%20%20at%20https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A85483%3A12%0A%20%20%20%20at%20https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87679%3A17%0A%20%20%20%20at%20ensure%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87600%3A38)%0A%20%20%20%20at%20module%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87677%3A14)%0A%20%20%20%20at%20https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A90313%3A22%0A%20%20%20%20at%20forEach%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A85820%3A20)%0A%20%20%20%20at%20loadModules%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A90297%3A5)%0A%20%20%20%20at%20createInjector%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A90219%3A19)%0A%20%20%20%20at%20doBootstrap%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87331%3A20)%0A%20%20%20%20at%20bootstrap%20(https%3A%2F%2Fspinnaker.ops.goromasamu.net%2Fvendor.bundle.js%3A87352%3A12)
    at http://myspinnaker.com/vendor.bundle.js:85483:12
    at http://myspinnaker.com/vendor.bundle.js:90337:15
    at forEach (http://myspinnaker.com/vendor.bundle.js:85820:20)
    at loadModules (http://myspinnaker.com/vendor.bundle.js:90297:5)
    at createInjector (http://myspinnaker.com/vendor.bundle.js:90219:19)
    at doBootstrap (http://myspinnaker.com/vendor.bundle.js:87331:20)
    at bootstrap (http://myspinnaker.com/vendor.bundle.js:87352:12)
    at angularInit (http://myspinnaker.com/vendor.bundle.js:87237:5)
    at HTMLDocument.<anonymous> (http://myspinnaker.com/vendor.bundle.js:118784:5)
    at fire (http://myspinnaker.com/vendor.bundle.js:169420:30)
```

**Comment by danielpeach** [comment]
Thanks - this was fixed in 1.4.1.

**Comment by rossedman** [comment]
@danielpeach thank you!
